schema.json

```diff
@@ -39,15 +21,35 @@
                         "$ref": "#/definitions/entity"
                     }
                 ]
-            }
+            },
+            "minItems": 1,
+            "type": "array",
+            "uniqueItems": true
         },
         "commit": {
             "$ref": "#/definitions/commit"
         },
+        "cff-version": {
+            "description": "The version of CFF used for providing the citation metadata.",
+            "examples": [
+                "1.2.0"
+            ],
+            "pattern": "^1\\.2\\.0$",
+            "type": "string"
+        },
+        "message": {
+            "description": "A message to the (human) reader of the file to let them know what to do with the citation metadata.",
+            "examples": [
+                "If you use this software, please cite it using the metadata from this file.",
+                "Please cite the paper from the references when using this software.",
```

Comment:
```suggestion
```
I suggest that this should go to keep a level of good practices support. The `references` section should only include references of the software that is described in `CITATION.cff`, i.e., work that it builds on (this could be papers or software dependencies or something else). If people want a paper to be cited, let them use `preferred-citation`.

Reply:
sounds good